### ISSUE TITLE
Replacement for `updated` last modified attribute?

### ISSUE THREAD

**Issue by MikeiLL** [burnash/gspread#562]
As far as I can tell worksheet.updated is depreciated, which makes sense because I think I saw that the API has changed in terms of getting modified date.

I had found [this old approach](https://github.com/burnash/gspread/issues/323) to checking if a sheet had been modified, but when I run on my sheet, `worksheet.updated` is always empty.

```
>>> chairs = client.open_by_url(url)
>>> sheet = chairs.get_worksheet(0)
>>> sheet.updated
>>>
```

Is there a new approach?

**Comment by omrivolk** [comment]
Would love a solution for this as well

**Comment by SweetBurlap** [comment]
The best that I can think of is to add a field to the spreadsheet with a last modified timestamp, then use a onEdit trigger in AppsScript to modify it - there's a few examples on stack overflow

**Comment by SweetBurlap** [comment]
Probably better than a custom field is to use the drive API to check for when it was lastModified

**Comment by alexislg2** [comment]
Hey. Have you guys found any clean solution to get the last modification time of the spreadsheet

This is very useful in my case: I don't want my script to do anything if a human user is still typing!

**Comment by alexislg2** [comment]
In my case I downgraded to gspread==0.6.2 and it works

**Comment by TobKed** [comment]
You can try to retrieve **modifiedTime** by [Google Drive API](https://developers.google.com/drive/api/v3/reference/files)

**Comment by lavigne958** [comment]
closed by #872 

one can now get the last modification time of a SpreadSheet using:  `sps.lastUpdateTime`

**Comment by sgrontflix** [comment]
I tried using this property and it gives me "KeyError" because it cannot find "modifiedTime". I have enabled the Google Sheet API and the Google Drive API, my credentials are correct and I added **spreadsheets** and **drive** to my scopes. What's going on?
